### DISCUSSION TITLE
How to create a new user?

### DISCUSSION THREAD

**Discussion by thx4nothing** [TomBursch/kitchenowl#67] [Q&A]
Hello, I am using the app for Android and want to create a new user, but don't know how. At the first start, it prompted me to create a user, but after that, I don't know where I can do that. When logged in, I can only change my password and stuff. When a new user downloads the app, it only shows the option to log in.

**Comment by TomBursch** [comment]
If you click on the bottom navigation bar on profile -> then on server (here you can find all server/admin settings) -> scroll down to the bottom -> next to user click on + and add a new user
